addons/l10n_be_edi/models/account_edi_format.py

```diff
@@ -46,18 +46,9 @@ def _export_efff(self, invoice):
         # Create file content.
         xml_content = b"<?xml version='1.0' encoding='UTF-8'?>"
         xml_content += self.env.ref('account_edi_ubl.export_ubl_invoice')._render(invoice._get_ubl_values())
-        vat = invoice.company_id.partner_id.commercial_partner_id.vat
-        xml_name = 'efff-%s%s%s.xml' % (vat or '', '-' if vat else '', invoice.name.replace('/', '_'))  # official naming convention
+        xml_name = invoice._get_efff_name('.xml')
```

Comment:
the `extension` parameter is weird since you always generate an `.xml` file. Why don't you hardcode the extension in the  `_get_efff_name` method or use `xml_name = '%s.xml' % invoice._get_efff_name()` instead?